client/src/components/Searchfield.css

```diff
@@ -0,0 +1,44 @@
+.searchfield {
+  position: relative;
+  margin: 0 auto;
+  height: 40px;
+  max-width: 640px;
+}
+
+.searchfield::before {
+  content: "";
+  position: absolute;
+  top: -5px;
+  height: 5px;
+  width: 100%;
+  background: var(--color-black-100);
+  border-radius: 5px;
+}
+
+.searchfield::after {
+  content: "";
+  position: absolute;
+  bottom: -5px;
+  height: 5px;
+  width: 100%;
+  background: var(--color-black-100);
+  border-radius: 5px;
+}
+
+.searchfield-button {

```

Comment:
BEM would be Searchfield__button. :)
as searchfield is also your component, you capitalize it.